doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -533,6 +539,9 @@ commands of Gallina. A sentence of the vernacular language, like in
 many natural languages, begins with a capital letter and ends with a
 dot.
 
+Sentences may be *decorated* with so-called *attributes*,
```

Comment:
You could use ```:token:`attributes` ``` here if you want that would add a direct link to their definition in the grammar.